polyfill/lib/temporal.mjs

```diff
@@ -1,12 +1,9 @@
-import { ES } from './ecmascript.mjs';
-
-export const Absolute = ES.GetIntrinsic('%Temporal.Absolute%');
-export const TimeZone = ES.GetIntrinsic('%Temporal.TimeZone%');
-export const DateTime = ES.GetIntrinsic('%Temporal.DateTime%');
-export const Date = ES.GetIntrinsic('%Temporal.Date%');
-export const YearMonth = ES.GetIntrinsic('%Temporal.YearMonth%');
-export const MonthDay = ES.GetIntrinsic('%Temporal.MonthDay%');
-export const Time = ES.GetIntrinsic('%Temporal.Time%');
-export const Duration = ES.GetIntrinsic('%Temporal.Duration%');
-
+export { Absolute } from './absolute.mjs';
+export { Date } from './date.mjs';
+export { DateTime } from './datetime.mjs';
+export { Duration } from './duration.mjs';
+export { MonthDay } from './monthday.mjs';
 export { now } from './now.mjs';
+export { Time } from './time.mjs';
+export { TimeZone } from './timezone.mjs';
+export { YearMonth } from './yearmonth.mjs';
```

Comment:
Just to be the devil's advocate here: is this any different from the original code? Is the only benefit of this that we don't need to import `intrinsticclass.mjs` anymore?

Reply:
To be honest, I'm not sure of the answer to that. That's why I was hoping @pipobscure would know why this file was using GetIntrinsic in the first place :smile: 

That said, I haven't been able to figure out a way in which it's _observably_ different; the polyfill still works, and it still works in bundled form in the browser console as well.